metamask.css

```diff
@@ -0,0 +1,26 @@
+
+body {
+  margin: 20px;
+}
+
+section {
+  margin: 20px 0 20px 0;
+}
+
+section > div.buttons {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.buttons > button.btn {
```

Comment:
This was the tangly part. I had to increase the specificy of this block by referencing a class and a parent class, to ensure it'd override the Bootstrap styles.